src/styles/header.scss

```diff
@@ -0,0 +1,69 @@
+:root {
+  --blue: #00acdc;
+}
+
+html {
+  --hover-color: var(--blue);
+
+  font-family: Roboto, system-ui;
+  font-weight: 500;
+  font-style: normal;
```

Comment:
Custom properties (variables) are usually defined in the `:root` selector for global scope. It's best to move the `--hover-color` property to the `:root` selector to maintain consistency and to ensure that it can be accessed from anywhere in the stylesheets.